build/build_system.bzl

```diff
@@ -235,3 +246,101 @@ def git_or_local_repository(name, branch, **kwargs):
             branch = branch,
             **kwargs
         )
+
+def _kong_install_impl(ctx):
+    outputs = []
+    strip_path = ctx.attr.strip_path
+    label_path = ctx.attr.src.label.workspace_name + "/" + ctx.attr.src.label.name
+    if not strip_path:
+        strip_path = label_path
+    prefix = ctx.attr.prefix
+    if prefix:
+        prefix = prefix + "/"
+
+    for file in ctx.files.src:
+        # skip top level directory
+        if file.short_path.endswith(label_path) or file.short_path.endswith(strip_path):
+            continue
+
+        # strip ../ from the path
+        path = "/".join(file.short_path.split("/")[1:])
```

Comment:
``` suggestion
        if file.short_path.startswith("../"): 
            path = "/".join(file.short_path.split("/")[1:])
            if path.startswith("../"):
                fail("Expected `%s` has only one `../` segment at the beginning" % file.short_path")
```

Handling more cases.

Reply:
I think we don't particualarly disallow `../../`, if even a rule outputs this. So I change this to:
```
        # strip ../ from the path
        path = file.short_path
        if file.short_path.startswith("../"):
            path = "/".join(file.short_path.split("/")[1:])
```

Reply:
good to me